utils.js

```diff
@@ -448,26 +429,31 @@ var Signals = class Signals extends Map {
             this.delete(object);
         }
     }
-}
+};
 
-var tweener = {
-    addTween(actor, params) {
+/**
+ * Note the name `Tweener` used previously was just a legacy name, we're actually using
+ * Widget.ease here.  This was renamed to avoid confusion with the deprecated `Tweener`
+ * moddule.
```

Comment:
```suggestion
 * module.
```

Reply:
@Lythenas - I like these changes.  Can you please do a PR (into this PR) with your suggested changes?

Will make it much easier for me.

Reply:
@Lythenas  - I've added most of these changes.  Can talk re the other points.